.editorconfig

```diff
@@ -0,0 +1,4 @@
+[*]
+indent_style = space
+tab_width = 4
+indent_size = 4
```

Comment:
EOF?

Reply:
I just kept it simple as this was the only requirement mentioned. Other things can be added later or I may add if it is good to do so.